db/schema.rb

```diff
@@ -17,10 +17,10 @@
   enable_extension "plpgsql"
 
   create_table "quotes", force: true do |t|
-    t.text     "text"
-    t.string   "author"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.text     "text",       null: false
+    t.string   "author",     null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
Great fix, now we'll never have created at dates that are null again? 

Can you add a test for this so that we can tell the schema changes have been picked up?